.github/workflows/create-release.yml

```diff
@@ -5,7 +5,14 @@ on:
     inputs:
       milestone_title:
         description: 'Milestone title (this release)'
-        required: true
+        required: true  # This does nothing! https://github.com/actions/runner/issues/1070
+      dry_run:
+        description: "Dry Run; if True, no release is made on the repo (though everything else is run!)"
+        type: choice
+        options:
+          - 'True'
+          - 'False'
+        default: 'True'
```

Comment:
Marvelous idea! `dry_run` is a fantastic idea, thank you for revealing the need for it. :hearts:

I also agree with the idea that `dry_run=True` should be the default value, and that the runner of the workflow needs to opt out of it to create a real release.